### DISCUSSION TITLE
Looking for good first time issues.

### DISCUSSION THREAD

**Discussion by meJevin** [Humanizr/Humanizer#1467] [General]
Hello, I'm interested in contributing to this library and helping as much as I can.

What issue would you suggest for a newcomer?

**Comment by meJevin** [comment]
Possibly, we could add issue labels like 'Up for grabs' or something like that.

**Comment by tranb3r** [comment]
There is already a "jump in" label

**Comment by recfab** [comment]
Hi, I was sent here by [CodeTriage](https://www.codetriage.com/humanizr/humanizer).

Looking at [`CONTIBUTING.md`](https://github.com/Humanizr/Humanizer/blob/main/CONTRIBUTING.md), I see that the `jump in` label is already listed there. 

Linking directly to the issue list already filtered by that label would be a nice-to-have, e.g. 

```text
... Some of the issues are labeled [`jump in`](https://github.com/Humanizr/Humanizer/labels/jump%20in). ...
```

I recommend a PR to linkify the label in the contributing guide, or close as won't do.
